mkdocs.yml

```
@@ -50,12 +37,25 @@ nav:
           - AMI Instructions and Guides:
             - Spreadsheet Formatting Overview: ami_spreadsheet_overview.md
             - Ingesting New Digital Objects and Collections using Spreadsheets or Google Sheets: AMIviaSpreadsheets.md
-            - Configuration for Google Sheets API: googleapi.md
+            - Linked Data Reconciliation: ami_lod_rec.md
             - Using the Islandora 7 Solr Importer: I7solrImporter.md
+            - Configuration for Google Sheets API: googleapi.md	  
```

```suggestion
            - Configuration for Google Sheets API: googleapi.md
```
Some extra spaces snuck in here and cause an error when run locally.